manifests/setup.pp

```diff
@@ -25,7 +26,9 @@
   katello_devel::bundle { 'install --without mysql:mysql2 --retry 3 --jobs 3 --path .vendor':
     environment => ['MAKEOPTS=-j'],
   } ->
-  katello_devel::bundle { 'exec npm install': } ->
+  katello_devel::bundle { 'exec npm install':
+    timeout => $npm_timeout,
```

Comment:
Test failure is relevant:

```
       Evaluation Error: Error while evaluating a Resource Statement, Katello_devel::Bundle[exec npm install]: has no parameter named 'timeout' (file: /home/travis/build/theforeman/puppet-katello_devel/spec/fixtures/modules/katello_devel/manifests/setup.pp, line: 29) on node travis-job-d9b6a10e-5bb7-499e-a206-ee586ce9ecf7.c.travis-ci-prod-2.internal
```

Looks like you'll need to add the parameter to https://github.com/theforeman/puppet-katello_devel/blob/master/manifests/bundle.pp